src/resolve/metadataResolver.ts

```diff
@@ -49,7 +49,10 @@ export class MetadataResolver {
       throw new TypeInferenceError('error_path_not_found', fsPath);
     }
 
-    this.forceIgnore = ForceIgnore.findAndCreate(fsPath);
+    // use the default ignore if we aren't using a real one
+    this.forceIgnore = this.useRealForceIgnore
```

Comment:
NIT: I don't love `real` in the var name. Especially given the fake mock registry used in tests. What do you think about `findForceIgnore` or `useDefaultForceignore`?